app/assets/javascripts/app/views/contacts_searcher_view.js

```diff
@@ -0,0 +1,122 @@
+app.views.ContactsSearcher = app.views.Base.extend({
```

Comment:
Actually, ContactTagSearch would be more acturate because it also search tags when used on the searchbar.